db_scripts/mosip_master/dml.sql

```diff
@@ -104,6 +104,9 @@ TRUNCATE TABLE master.template cascade ;
 
 \COPY master.template (id,name,descr,file_format_code,model,file_txt,module_id,module_name,template_typ_code,lang_code,is_active,cr_by,cr_dtimes) FROM './dml/master-template.csv' delimiter ',' HEADER  csv;
 
+\COPY master.blocklisted_words (word,descr,lang_code,is_active,cr_by,cr_dtimes) FROM './dml/master-blacklisted_words.csv' delimiter ',' HEADER  csv;
```

Comment:
Blocklisted words are country-specific data it should not be part of DML scripts.

Reply:
removed and as we have already added in mosip-data a part of merging back to develop so pr is not required for mosip-data